src/sql_engine/src/query/mod.rs

```diff
@@ -12,15 +12,20 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+use std::convert::TryFrom;
+
+use sqlparser::ast::ObjectName;
+
+use sql_types::SqlType;
+
+pub mod bind;
 ///! Module for representing how a query will be parameters bound, executed and
 ///! values represented during runtime.
```

Comment:
Weird formatter behaviour, this is a doc for `bind` module for some reason it wasn't rearranged with it.